README.md

```diff
@@ -34,7 +34,7 @@ For more information on the importance of a professional README for your reposit
   <img src="download.jpeg" alt="logo" width="140"  height="auto" />
   <br/>
 
```

Comment:
- [ ] Since you have removed the `Live Demo` section from your readme file, also remove it from your table of contents so as to avoid broken links for a more professional readme file
![image](https://github.com/gilberthappi/Portfolio_setup_and_mobile_first/assets/81932478/bcfb4ee8-a430-45b1-941c-d39ecb4ca10b)

This applies to the Deployment and Run Tests links as well
![image](https://github.com/gilberthappi/Portfolio_setup_and_mobile_first/assets/81932478/98b4bcb3-c90c-439d-98ea-f702a82b1532)

Reply:
Okey
Thanks